### DISCUSSION TITLE
Предложение добавить в документацию

### DISCUSSION THREAD

**Discussion by M3teora** [totumonline/totum-ru-issues-and-discussions#528] [Есть идея, а что если...]
Добрый день. Не мог найти по документации данную функцию, повезло, что очень случайно набрёл. Обычно я ищу это как "число прописью", но по такому запросу поиск по документации ничего не выдал. Предложение добавить слово "прописью" в поиск) 

<img width="998" alt="image" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/7896517/09737525-57ca-4501-b232-93f6f3cb6c7e">

**Comment by ttm-alexeypolunin** [comment]
Да, буду обновлять доки - напишу там. С ней косяк такой, что она только в русской озвучке работает :)
